pkg/controllers/uiplugin/logging.go

```diff
@@ -62,6 +62,16 @@ func createLoggingPluginInfo(plugin *uiv1alpha1.UIPlugin, namespace, name, image
 					Port:      8080,
 				},
 			},
+			{
```

Comment:
Is it ok to create the proxy entry even when the korrel8r plugin isn't installed?

Reply:
I think is ok as the console proxy just allows the plugin to communicate to the in cluster service. If korrel8r deployment fails, or is not there, requests will fail and the plugin won't show the correlation links.